README.md

```diff
@@ -80,13 +81,14 @@ DESCRIPTION:
 OPTIONS:
    --host value        --host localhost:6060 (default: "http://localhost:6060") [$OLS_TEST_HOST]
    --authtoken value   --authtoken authtoken [$OLS_TEST_AUTH_TOKEN]
-   --uuid value        --uuid f519d9b2-aa62-44ab-9ce8-4156b712f6d2 (default: "08fd0205-d2d8-4954-b83e-86226dd4e2ac") [$OLS_TEST_UUID]
+   --uuid value        --uuid f519d9b2-aa62-44ab-9ce8-4156b712f6d2 (default: "76d8f64d-2bf3-49ac-82c3-22011ddc2284") [$OLS_TEST_UUID]
    --duration value    --duration 1m (default: 1m0s) [$OLS_TEST_DURATION]
    --workers value     --workers 10 (default: 10) [$OLS_TEST_WORKERS]
    --eshost value      --eshost eshosturl [$OLS_TEST_ES_HOST]
    --esindex value     --esindex esindex [$OLS_TEST_ES_INDEX]
    --metricstep value  --metricstep 30 (default: 30) [$OLS_TEST_METRIC_STEP]
    --profiles value    --profiles metrics.yaml,metrics-report.yaml (default: "attacker/assets/profiles/metrics-report.yaml,attacker/assets/profiles/metrics-timeseries.yaml") [$OLS_TEST_PROFILES]
+   --queryonly         --query (default: false) [$OLS_QUERY_ONLY]
```

Comment:
$OLS_TEST_QUERY_ONLY ?

Reply:
I had this flag as an option for someone to be able to run load test only on `/v1/query` endpoints as full list of apis might not be required all the times.

Reply:
I meant to say, did you mean to say [$OLS_TEST_QUERY_ONLY] here?

Reply:
Changed as suggested.